universal-application-tool-0.0.1/app/forms/BlockForm.java

```diff
@@ -1,5 +1,6 @@
 package forms;
 
+/** Form for updating a screen (block) of a program. */
```

Comment:
suggestion: (this always confuses me each time I see this form)
```suggestion
/** Form for updating the name and description of a screen (block) of a program. */
```

Reply:
Great suggestion!